config/routes.rb

```diff
@@ -1,6 +1,10 @@
-Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+Rails.application.routes.draw do 
+  # get "users/", to: "users#index"
+  # get "users/:id", to: "users#show"
+  # get "users/:user_id/posts", to: "posts#index"
+  # get "users/:user_id/posts/:id", to: "posts#show"
```

Comment:
- [ ] (Optional) Kindly remove all unnecessary codes. Removing all unnecessary codes shows that you adopt clean and DRY principles when writing your code. 👍